### DISCUSSION TITLE
[Desktop - Photos] Auto-update on Linux (DEB specifically)?  What about Auth?

### DISCUSSION THREAD

**Discussion by taivlam** [ente-io/ente#4276] [Q&A] [answered]
According to the [general FAQ](https://help.ente.io/photos/faq/general#how-do-i-ensure-that-the-ente-desktop-app-stays-up-to-date-on-my-system) and the Download [page](https://ente.io/download) under Linux, the latter which says:
> Linux AppImage supports auto updates.
> We also provide auto updating [RPM / DEB packages](https://github.com/ente-io/photos-desktop/releases/latest).

Ente Photos is documented to have auto updates built into the application.  I have some questions, regarding the DEB binary:
* Does this mean the fact that I packaged Ente Photos on the MPR (for Debian-/Untubu-based Linux distros) is redundant?
* If I were to install the DEB manually, will the application update itself via APT if I run `$ sudo apt update && sudo apt upgrade` afterwards?
* On a related tangent, does the DEB for Ente Auth also auto-update?

**Comment by mnvr** [comment, marked as answer]
Re the photos bit - we use the default auto update configuration provided by [Electron builder](https://www.electron.build/auto-update#auto-updatable-targets), so their documentation / [GitHub](https://github.com/electron-userland/electron-builder) might have the answers for q1/q2.

Re auth q3, currently it does not autoupdate.
